### PR TITLE
Add Limbo egg

### PR DESCRIPTION
**Pull request by xMikux**
# Description

Another Limbo from [LOOHP](https://github.com/LOOHP/Limbo).

## Checklist for all submissions

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you tested and reviewed your changes with confidence that everything works?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

## New egg Submissions

1. [x] Does your submission pass tests (server is connectable)?
2. [ ] Does your egg use a custom docker image?
    * [ ] Have you tried to use a generic image?
    * [ ] Did you PR the necessary changes to make it work?
3. [x] Have you added the egg to the main README.md and any other README files in subdirectories of the egg (e.g /game_eggs) according to the alphabetical order?
4. [x] Have you added a unique README.md for the egg you are adding according to the alphabetical order?
5. [x] You verify that the start command applied does not use a shell script
    * [ ] If some script is needed then it is part of a current yolk or a PR to add one
6. [x] The egg was exported from the panel

### CODE REVIEW

**Comment by parkervcp** [comment]
> Another Limbo from [LOOHP](https://github.com/LOOHP/Limbo).

There is already a limbo server. Is there a reason to add another?

**Comment by xMikux** [comment]
Yes, these two are slightly different.
And I see somewhere that still has someone saying about Limbo eggs, so it will be good to have one.

**Comment by QuintenQVD0** [comment]
if  you add it to all needed README.md then this can be merged

**Comment by xMikux** [comment]
Hi, I updated the missing readme.

File: game_eggs/minecraft/java/limbo/egg-limbo.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "startup": "java -Xms128M -Xmx{{SERVER_MEMORY}}M -jar {{SERVER_JARFILE}}",
+    "config": {
+        "files": "{\r\n    \"server.properties\": {\r\n        \"parser\": \"properties\",\r\n        \"find\": {\r\n            \"server-ip\": \"0.0.0.0\",\r\n            \"server-port\": \"{{server.build.default.port}}\"\r\n        }\r\n    }\r\n}",
+        "startup": "{\r\n    \"done\": \"Limbo server listening on \"\r\n}",
+        "logs": "{}",
+        "stop": "stop"
+    },
+    "scripts": {
+        "installation": {
+            "script": "#!\/bin\/ash\r\n# Limbo Installation Script\r\n#\r\n# Server Files: \/mnt\/server\r\nLATEST_LIMBO_NAME=$(curl -s https:\/\/ci.loohpjames.com\/job\/Limbo\/lastSuccessfulBuild\/api\/json | jq -r .artifacts[1].displayPath)\r\ncd \/mnt\/server\r\necho \"Downloading Limbo server jar...\"\r\ncurl -o Limbo.jar https:\/\/ci.loohpjames.com\/job\/Limbo\/lastSuccessfulBuild\/artifact\/target\/$LATEST_LIMBO_NAME\r\nif [ ! -f server.properties ]; then\r\necho -e \"Downloading Limbo server.properties\"\r\ncurl -o server.properties https:\/\/raw.githubusercontent.com\/LOOHP\/Limbo\/master\/src\/main\/resources\/server.properties\r\nfi",
```
Comment (QuintenQVD0) [inline game_eggs/minecraft/java/limbo/egg-limbo.json:25]: please also add here the logic to rename to old .jar to a .jar.old for if they hit reinstall + you don't use here the {{SERVER_JARFILE}} variable but hard coded it:

```ash
#!/bin/ash
# Limbo Installation Script
#
# Server Files: /mnt/server
mkdir -p /mnt/server

LATEST_LIMBO_NAME=$(curl -s https://ci.loohpjames.com/job/Limbo/lastSuccessfulBuild/api/json | jq -r .artifacts[1].displayPath)

cd /mnt/server

if [ -f ${SERVER_JARFILE} ]; then
	mv ${SERVER_JARFILE} ${SERVER_JARFILE}.old
fi

echo "Downloading Limbo server jar..."
curl -o ${SERVER_JARFILE} https://ci.loohpjames.com/job/Limbo/lastSuccessfulBuild/artifact/target/$LATEST_LIMBO_NAME


if [ ! -f server.properties ]; then
    echo -e "Downloading Limbo server.properties"
    curl -o server.properties https://raw.githubusercontent.com/LOOHP/Limbo/master/src/main/resources/server.properties
fi
```

File: game_eggs/minecraft/java/limbo/egg-limbo.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    "exported_at": "2023-02-10T19:28:32+08:00",
+    "name": "Limbo",
+    "author": "xEfinax@protonmail.com",
+    "description": "Standalone server program Limbo.",
+    "features": null,
+    "docker_images": {
+        "ghcr.io\/pterodactyl\/yolks:java_17": "ghcr.io\/pterodactyl\/yolks:java_17"
+    },
+    "file_denylist": [],
+    "startup": "java -Xms128M -Xmx{{SERVER_MEMORY}}M -jar {{SERVER_JARFILE}}",
```
Comment (QuintenQVD0) [inline game_eggs/minecraft/java/limbo/egg-limbo.json:16]: Please use here: `java -Xms128M -XX:MaxRAMPercentage=95.0  -jar {{SERVER_JARFILE}}

**Review by QuintenQVD0 — CHANGES_REQUESTED** [review]

**Comment by xMikux** [comment]
I updated the requested changes script and startup command.

**Comment by QuintenQVD0** [comment]
> 

It still hardlocks the download .jar name

**Comment by xMikux** [comment]
Fixed

**Review by QuintenQVD0 — APPROVED** [review]

**Comment by QuintenQVD0** [comment]
LGTM!

**Pull request merged**